nixos/modules/misc/ids.nix

```diff
@@ -339,6 +339,7 @@
       rss2email = 312;
       cockroachdb = 313;
       zoneminder = 314;
+      prometheus2 = 315;
```

Comment:
One user for both prometheus instances would be enough. Especially because we are going to remove prometheus1 soon.

Reply:
I agree.